packages/babel-standalone/test/preset-stage-1.test.js

```diff
@@ -9,6 +9,7 @@ describe("stage-1 preset", () => {
 
   it("should parser decimal literal", () => {
     const output = Babel.transform("0.3m", {
+      configFile: false,
```

Comment:
Babel standalone should have the "browser" files of `@babel/core`, and not support config loading at all 🤔

Reply:
Yeah, it seems that `@babel/plugin-transform-export-namespace-from@7` is loaded, but its dependency of the syntax plugin is somehow not fulfilled.